backend/src/test/java/de/amos/apachepulsarui/controller/TopicControllerTest.java

```diff
@@ -77,4 +83,35 @@ void returnTopicByName() throws Exception {
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.name", equalTo(fullTopic)));
     }
+
+    @Test
+    void getSubscriptionByNameAndTopic() throws Exception {
+        String subscription = "R2D2";
+        String topic = "Droide";
```

Comment:
Maybe you could use a real topicName here, including the // . Then the test would fail due to the pathParameter in the controller.